pkg/acceptance/bettertestspoc/assert/objectassert/task_snowflake_gen.go

```diff
@@ -107,11 +107,17 @@ func (t *TaskAssert) HasComment(expected string) *TaskAssert {
 	return t
 }
 
-func (t *TaskAssert) HasWarehouse(expected string) *TaskAssert {
+func (t *TaskAssert) HasWarehouse(expected *sdk.AccountObjectIdentifier) *TaskAssert {
```

Comment:
We should not modify the generated assertions: if you need a better one (because this one is bad for some reason), then please document it and for now add other assertion in the _ext/go file/

Reply:
Re-generated the asserts and removed what was not compiling and the needed asserts modified by hand were moved to _ect.go